index.ts

```diff
@@ -68,7 +73,11 @@ async function loop() {
   // Incremental loop to check the balance
   let newBalance = undefined;
   for (let i = 1; i <= config.MAX_ATTEMPTS; i++) {
-    newBalance = await getBalance(data);
+    try {
+      newBalance = await getBalance(data);
+    } catch (e) {
+      throw new AlertError(`${e}`)
```

Comment:
When you create an error with message `test` for example, it will automatically format it this way: `Error: test` so to avoid that you can write: `new AlertError(e.message)`